### ISSUE TITLE
SceneWriter.py: light_node.sb_ysize issue

### ISSUE THREAD

**Issue by consultit** [tobspr/Panda3D-Bam-Exporter#10]
In SceneWriter.py (line 156) there is an auto assignment:
```
light_node.sb_ysize = light_node.sb_ysize
```
I think it should be:
```
light_node.sb_ysize = light_node.sb_xsize
```
bye

**Comment by tobspr** [comment]
You are right, seems theres a typo, thanks! Feel free to make a PR :)

**Comment by rdb** [comment]
Closed in #12.
